bin.js

```diff
@@ -31,3 +34,12 @@ function getConfig() {
   }
 }
 
+function getRules(set) {
+  var configFileName = set ? ('eslint-config-' + set) : null
```

Comment:
> Installing eslint-find-new-rules globally and would not work, as referenced configs couldn't be found.

I'm not a fan of installing tools like this globally anyway, so that's fine.

What I'm worried about though is that doing this will utilize the `~/.eslintrc` file which could contain extra rules (unless I'm mistaken). Also, in projects like [mine](https://github.com/kentcdodds/eslint-config-kentcdodds) where there are multiple files representing eslint config, I worry that having an `.eslintrc` file in there would be a problem... Am I misunderstanding it?

I was thinking a way to get around this would be to copy the config to a temporary directory on the filesystem and doing the check there... Thoughts?